torch/utils/benchmark/utils/timer.py

```diff
@@ -172,6 +179,7 @@ def __init__(
         self,
         stmt: str = "pass",
         setup: str = "pass",
+        global_setup: Optional[str] = None,
```

Comment:
very minor nit - why do you need global_setup as Optional[str] instead of str = ""? That's what it ends up being anyway?

Reply:
That's a good point. I'm probably getting overzealous with optionals when here the null state actually is empty string.